demos/src/scss/resized.scss

```diff
@@ -9,5 +9,5 @@ $o-grid-gutters: (default: 5px);
 @import 'common';
 
 .test body:after {
-	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: #{$o-grid-gutters}';
+	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: 5px';
```

Comment:
I'm not sure I follow. How will it break? And how are the breakpoint values being hardcoded ok, but the gutter not?